client/src/components/CreateOrUpdateForm/CreateOrUpdateForm.js

```diff
@@ -326,19 +326,49 @@ export default class CreateOrUpdateForm extends React.Component {
                 latitude.isValid = false;
                 latitude.message = 'Latitude must not be blank';
                 formIsValid = false;
+            } else if (latitude.value.includes('°')) {
+                latitude.isValid = false;
+                latitude.message = 'Please use decimal coordinates, not degrees. ' +
+                    'To convert, input your degrees into Google Maps and copy the new numbers here.';
+                formIsValid = false;
             } else if (!validator.matches(latitude.value, latitudeRegex)) {
                 latitude.isValid = false;
                 latitude.message = 'Latitude must be valid';
                 formIsValid = false;
+            } else {
+                const latAsDouble = parseFloat(latitude.value);
+                // Alaska is the furthest north location and its latitude is approximately 71
+                // The American Samoa is the furthest south location and its latitude is approximately -14
+                if (latAsDouble > 72 || latAsDouble < -15) {
+                    latitude.isValid = false;
+                    latitude.message = 'Latitude is not near the United States';
+                    formIsValid = false;
+                }
             }
+
             if (validator.isEmpty(longitude.value)) {
                 longitude.isValid = false;
                 longitude.message = 'Longitude must not be blank';
                 formIsValid = false;
-            } else if (!validator.matches(longitude.value, longitudeRegex)) {
+            } else if (longitude.value.includes('°')) {
+                longitude.isValid = false;
+                longitude.message = 'Please use decimal coordinates, not degrees. ' +
+                    'To convert, input your degrees into Google Maps and copy the new numbers here.';
+                formIsValid = false;
+            }
+            else if (!validator.matches(longitude.value, longitudeRegex)) {
                 longitude.isValid = false;
                 longitude.message = 'Longitude must be valid';
                 formIsValid = false;
+            } else {
+                const lonAsDouble = parseFloat(longitude.value);
+                // Guam is the furthest west location and its longitude is approximately 144
+                // Puerto Rico is the furthest east location and its longitude is approximately -65
+                if (lonAsDouble > -64 && !(lonAsDouble < 180 && lonAsDouble > 143)) {
```

Comment:
@benvogler Same here

Reply:
Damn prime meridian making this boolean logic all confusing. I think this is right